allauth/headless/app_settings.py

```diff
@@ -27,6 +27,10 @@ def TOKEN_STRATEGY(self):
     def FRONTEND_URLS(self):
         return self._setting("FRONTEND_URLS", {})
 
+    @property
+    def INPUTS(self):
+        return self._setting("INPUTS", {})
```

Comment:
These class overrides need to inherit from `allauth.headless.inputs`, we can document that.

Reply:
Here's an example I'm using locally:

```python
from allauth.headless.account.inputs import RequestPasswordResetInput
from django.http import HttpRequest

from cloud_console.django.apps.webauth.models import User


class RequestPasswordResetInput(RequestPasswordResetInput):
    def save(self, request: HttpRequest, **kwargs):
        # Only users w/ passwords can request to reset their password
        if self.users:
            email = self.cleaned_data["email"]
            try:
                user = User.objects.get(email=email)
            except User.DoesNotExist:
                return email
            else:
                if not user.has_usable_password():
                    return email
        return super().save(request, **kwargs)
```

Here is how I configure my settings:

```python
HEADLESS_INPUTS = {
    "reset_password": "cloud_console.django.apps.webauth.inputs.RequestPasswordResetInput"
}
```